Sources/FirebladeECS/FamilyProtocols.swift

```diff
@@ -23,13 +23,13 @@ public protocol FamilyRequirementsManaging {
 }
 
 public protocol FamilyEncoding: FamilyRequirementsManaging {
-    static func encode(components: [Components], into container: inout UnkeyedEncodingContainer, using strategy: CodingStrategy) throws
+    static func encode(componentsArray: [Components], into container: inout UnkeyedEncodingContainer, using strategy: CodingStrategy) throws
```

Comment:
Parent1 ... Parent5 correspond to these methods and protocol, and they are not updated - thus still seeing the error. 😞